### PR TITLE
fix: add Gurjeet Singh Virdee

### PR DESCRIPTION
**Pull request by gurjeetsinghvirdee**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/884"></a>

### CODE REVIEW

File: public/data/gurjeetsinghvirdee.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      },
+      {
+        "title": "Started Technical Content Writing",
+        "date": "October 2020",
+        "description": "Writing Technical Articles in GeeksforGeeks",
+        "icon": "book",
+        "color": "red"
+      }
+    ]
+  }
```
Comment (Panquesito7) [inline public/data/gurjeetsinghvirdee.json:34]: Missing an endline.
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: public/data/gurjeetsinghvirdee.json [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+        "name": "Add me on LinkedIn",
+        "url": "https://www.linkedin.com/in/gurjeet-singh-virdee-25a476199/",
+        "icon": "linkedin"
+      }
+    ],
+    "milestones": [
+      {
+        "title": "Started Open Source Contribuitions",
+        "date": "Sep 2020",
+        "description": "Started some beginner freind open source projects",
```
Comment (syedsohan) [inline public/data/gurjeetsinghvirdee.json:22]:
```suggestion
        "description": "Started some beginner-friendly open source projects",
```

**Review by syedsohan — CHANGES_REQUESTED** [review]

File: public/data/gurjeetsinghvirdee.json (hunk 2 above)
Comment (gurjeetsinghvirdee) [reply public/data/gurjeetsinghvirdee.json]: Thankyou for your feedback! I edited the file

**Review by syedsohan — APPROVED** [review]

File: public/data/gurjeetsinghvirdee.json (hunk 1 above)
Comment (gurjeetsinghvirdee) [reply public/data/gurjeetsinghvirdee.json]: Thankyou for your feedback! I changed the alignment

**Review by Panquesito7 — APPROVED** [review]
LGTM 🚀 Thank you for your contribution! 😄👍

**Pull request merged**